R/fullREMIND.R

```diff
@@ -137,7 +137,7 @@ fullREMIND <- function() {
   system(paste0('sed -i "/dummy/d" ', getConfig()$outputfolder, "/p37_clinker-to-cement-ratio.cs3r"))
 
   calcOutput("Capacity", subtype = "capacityByTech",                   round = 6,  file = "pm_histCap.cs3r")
-  calcOutput("Capacity", subtype = "capacityByTech_windoff",           round = 6,  file = "pm_histCap_windoff.cs3r")
+  calcOutput("Capacity", subtype = "capacityByTech",                   round = 6,  file = "pm_histCap_windoff.cs3r")  # to remove after updating remind
```

Comment:
If the corresponding REMIND update is happening quasi at the same time, there is no need for this (as, you would increment the inputdata version in REMIND). If not, please don't forget to clean this up once you merged your REMIND changes :)